### PR TITLE
Add SQLAlchemy models and wave 2 ingestion

### PR DESCRIPTION
**Pull request by hsolleder**

### CODE REVIEW

**Comment by pierreleripoll** [comment]
@ymarcon I need your help here :superhero:

File: backend/__init__.py
Comment (ymarcon) [inline backend/__init__.py:1]: Do not commit the __init__.py files.

File: backend/pyproject.toml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 readme = "README.md"
+package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
-pre-commit = "^3.7.1"
+sqlalchemy = "^2.0.32"
+python-dotenv = "^1.0.1"
+psycopg2-binary = "^2.9.9"
+sqlmodel = "^0.0.21"
```
Comment (ymarcon) [inline backend/pyproject.toml:12]: 0.0.22 is out :)

Comment (hsolleder) [reply backend/pyproject.toml:12]: Thanks! Bumped to 0.0.22 in a0da18e

File: backend/__init__.py
Comment (hsolleder) [reply backend/__init__.py:1]: Removed in 2bc0707

File: backend/database/create_tables.py
[the remark this answers is not in the document]
Comment (hsolleder) [reply backend/database/create_tables.py:1]: I created a new issue (#4 ) to tackle this important aspect in the near future.

**Review by ymarcon — APPROVED** [review]

**Pull request merged**